src/Controller/HearingController.php

```diff
@@ -417,11 +424,59 @@ public function hearingPostApprove(CaseEntity $case, HearingPostResponse $hearin
             $this->entityManager->persist($document);
 
             $digitalPostHelper->createDigitalPost($document, $documentTitle, get_class($case), $case->getId(), [], $digitalPostRecipients);
+
+            // Create digital post case event (sagshændelse)
+            $digitalPostCaseEvent = new CaseEvent();
+            $digitalPostCaseEvent->setCaseEntity($case);
+            $digitalPostCaseEvent->setCategory(CaseEvent::CATEGORY_OUTGOING);
+
+            $caseEventPartyRelation = new CaseEventPartyRelation();
+            $caseEventPartyRelation->setParty($sender);
+            $caseEventPartyRelation->setCaseEvent($digitalPostCaseEvent);
+            $caseEventPartyRelation->setType(CaseEventPartyRelation::TYPE_RECIPIENT);
+            $this->entityManager->persist($caseEventPartyRelation);
+
+            $digitalPostCaseEvent->setSubject(CaseEvent::SUBJECT_HEARING_CONTRADICTIONS_BRIEFING);
+            $digitalPostCaseEvent->setReceivedAt(new DateTime('now'));
+            $digitalPostCaseEvent->setCreatedBy($caseEventUser);
+
+            $digitalPost = $digitalPostRepository->findByDocumentAndCase($document, $case);
```

Comment:
Can these setter calls be moved up to the other setter calls on `$digitalPostCaseEvent`?